src/views/recipeView.js

```diff
@@ -104,7 +105,35 @@ ${recipe.ingredients
 </div>`;
   recipeContainer.innerHTML = '';
   recipeContainer.insertAdjacentHTML('afterbegin', html);
+  const btnIncrease = document.querySelector('.btn--increase-servings');
+  const btnDecrease = document.querySelector('.btn--decrease-servings');
+  btnIncrease.addEventListener('click', btnIncreaseHandler);
+  btnDecrease.addEventListener('click', btnDecreaseHandler);
+  btnDecrease.disabled = servings === 1 ? true : false;
```

Comment:
make it as a function, like initiateHandler or something

Reply:
okay